Add and document purgeCssEnabled option (#20)
lucleray/next-purgecss: readme.md

## Options

### `purgeCssEnabled`

By default, `next-purgecss` will always remove unused CSS, regardless of build environment. You can change that by defining a function for the `purgeCssEnabled` option. The `purgeCssEnabled` function receives two arguments:

| Argument | Type | Description |
| --- | --- | --- |

> | `dev` | `Boolean` | Shows if the compilation is done in development mode (`true`) or production mode (`false`) |

Reviewer: ```suggestion
| `dev` | `Boolean` | `true` in development mode (running `next`) or `false` in production mode (running `next start`) |
```

> | `isServer` | `Boolean` | Shows if the configuration will be used for server side compilation (`true`), or client side compilation (`false`) |

Reviewer: I think we can make this shorter:

```suggestion
| `isServer` | `Boolean` | `true` during server side compilation or `false` during client side compilation |
```
